### DISCUSSION TITLE
Is there an electrostatics module?

### DISCUSSION THREAD

**Discussion by adamdwexler** [idaholab/moose#16436] [Q&A General] [answered]
Hi All,

I'm new to the MOOSE environment and trying to get the lay of the land. I am working on modeling an electrohydrodynamic multiphase flow with heat exchange. Can anyone suggest a good module to build the EHD forces in? Tensor mechanics perhaps?

Thanks for the assistance!
Adam

**Comment by aeslaughter** [comment, marked as answer]
If you want to understand how to develop an application, then I suggest you follow our tutorial: https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/index.html. It is a work-in-progress, but there is enough there to help you understand the big picture in creating an application.

The modules pages also have a lot of information that could help you understand our current capabilities: https://mooseframework.inl.gov/modules/index.html

**Reply by adamdwexler** [reply]
Thank you Andrew for directing me towards these tutorials. This is a tremendous ecosystem to get started in and I just discovered the summer workshop. Excited to learn!

**Comment by mangerij** [comment]
Yeah, setting up a Poisson problem in MOOSE is pretty straightforward and I think it's an excellent exercise for a beginner. We solve the magnetostatic/electrostatic Poisson problems in Ferret ( https://mangerij.github.io/ferret/ ). Since magnetic/electric charge density on the RHS of the Poisson equation can be different depending on your physics, it probably makes sense for you to make your own app.

I gave a cursory glance at EHD models and it seems the Navier-Stokes module could be useful for you as well so I would suggest for you to take a look there: moose/modules/navier_stokes

**Reply by adamdwexler** [reply]
Hi John,
Thank you for the encouraging comment. Your work on ferroic nanostructure seems really interesting! Good call on including Navier-Stokes and I think I will take a look at Ferret for inspiration.
